src/compiler/checker.ts

```diff
@@ -26825,6 +26825,11 @@ namespace ts {
                                 case SyntaxKind.GreaterThanGreaterThanGreaterThanToken:
                                 case SyntaxKind.GreaterThanGreaterThanGreaterThanEqualsToken:
                                     reportOperatorError();
+                                    break;
+                                case SyntaxKind.AsteriskAsteriskToken:

```

Comment:
This error should be on `SyntaxKind.AsteriskAsteriskEqualsToken` as well. So the following code should not be allowed:

```ts
a **= 2;
```